### PR TITLE
Added the phone and email validation

### PR DESCRIPTION
**Pull request by Daliego**

### CODE REVIEW

File: create-tables/tables.sql [hunk 1]
```diff
@@ -44,9 +44,10 @@ CREATE TABLE PEDIDO (
 	ID_METODO_PAGAMENTO INT NOT NULL REFERENCES METODO_DE_PAGAMENTO,
 	ID_PARCEIRO INT NOT NULL REFERENCES PARCEIRO,
 	ID_ANALISTA INT NOT NULL REFERENCES ANALISTA,
-	VALOR_COMISSAO FLOAT DEFAULT 0.0,
 	DATA_VENCIMENTO_PAGAMENTO_COMISSAO TIMESTAMP NOT NULL,
 	DATA_PAGAMENTO_COMISSAO TIMESTAMP,
+	VALOR_COMISSAO FLOAT DEFAULT 0.0,
+	VALOR_ALUDUEL FLOAT DEFAULT 0.0,
```
Comment (rm3d0nc4) [inline create-tables/tables.sql]: Não entendi o porque dessa linha

**Review by rm3d0nc4 — CHANGES_REQUESTED** [review]

**Comment by Daliego** [comment]
Accept me now please

**Review by Daliego — COMMENTED** [review]
accept me now!!!

**Review by rm3d0nc4 — APPROVED** [review]
Congratulations! You pull request has approved

**Pull request merged**